keg_elements/crypto.py

```diff
@@ -0,0 +1,47 @@
+import os
+
+import cryptography.hazmat.primitives.ciphers as ciphers
+from cryptography.hazmat.backends import default_backend
+
+
+def aes_cipher(key, iv, mode=None):
```

Comment:
For testing it makes sense to be able to pass in the `iv`, but in general, I think it should default to `None` and we should set it.